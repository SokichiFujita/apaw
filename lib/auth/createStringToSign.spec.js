const createStringToSign = require('./createStringToSign');

describe('createStringToSign', () => {
  it('should create a formatted string of various request values', () => {
    const stringToSign = createStringToSign({
      algorithm: 'algorithm',
      canonicalRequest: 'canonicalRequest',
      longDate: '20200308T123600Z',
      region: 'region',
      service: 'service',
      shortDate: '20200308',
      terminationString: 'terminationString',
    });
    expect(stringToSign).toMatchSnapshot();
  });
});
