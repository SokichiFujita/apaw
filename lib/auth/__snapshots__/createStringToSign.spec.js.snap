// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createStringToSign should create a formatted string of various request values 1`] = `
"algorithm
20200308T123600Z
20200308/region/service/terminationString
canonicalRequest"
`;
